### PR TITLE
feat: migrate `/game/{game}/comments` to React

### PR DESCRIPTION
**Pull request by wescopeland**
This PR migrates the `/game/{game}/comments` route from Blade to React.

The UI has very nearly 100% code coverage. There should be only a few functional changes.

https://github.com/user-attachments/assets/b34d6ba6-1358-49f3-be4e-767732b55eaa

Behaviors that I've verified:
* Unauthenticated users are prompted to sign in and not shown a textarea.
* The textarea still auto-expands and validates against input size.
* Submitting a comment sends the user to the correct page.
* Deleting a comment sends the user to the correct page.
* Submitting a comment for the first time to a game wall subscribes the user to that game wall.
* The user can still toggle their subscription.
* The user can paginate, and paginations are client-side navigations.
* Muted users are not shown a textarea and are informed of their mute status.

Some new behaviors worth noting:
* The textarea's placeholder text has been slightly modified.
* Previously, banned user comments were removed by PHP template code. Now they're filtered out at the controller level, based on user privileges to actually see them.
* Submitting a comment will always take the user to the correct page on submit.
* Hovering over timestamps now uses a proper `<BaseTooltip />` component (forum recent posts is also affected by this change).
* User absolute timestamp preferences are now respected.

**Next steps:**
* Migrate the other comment pages.

### CODE REVIEW

File: app/Community/Controllers/GameCommentController.php [hunk 1]
```diff
@@ -4,23 +4,69 @@
...
+            game: GameData::fromGame($game)->include('badgeUrl', 'system'),
+            paginatedComments: PaginatedData::fromLengthAwarePaginator(
+                $paginatedComments,
+                total: $paginatedComments->total(),
+                items: CommentData::fromCollection($paginatedComments->getCollection())
+            ),
+            subscription: $subscription ? SubscriptionData::from($subscription) : null,
+        );
+
+        return Inertia::render('game/[game]/comments', $props);
```
Comment (Jamiras) [inline app/Community/Controllers/GameCommentController.php:69]: Not sure where the page title is coming from, but it lost a colon.

Before:
image

After:
image

But maybe it should be game title first (supported game files looks like it's not rendering correctly):
image

Comment (wescopeland) [reply app/Community/Controllers/GameCommentController.php:69]: Good catch on both issues. Fixed in latest.

File: resources/js/pages/game/[game]/comments.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ const GameComments: AppPage<App.Community.Data.GameCommentsPageProps> = ({ game
 
   return (
     <>
-      <Head title={t('Comments: :game', { game: game.title })}>
+      <Head title={t('Comments: :gameTitle', { gameTitle: game.title })}>
```
Comment (Jamiras) [inline resources/js/pages/game/[game]/comments.tsx:13]: Suggest `Comments - :gameTitle` for consistency with Supported Game Files, Manage Hashes, and Manage Claims pages.

Comment (wescopeland) [reply resources/js/pages/game/[game]/comments.tsx]: No problem, updated in latest.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**